pkg/config/config.go

```diff
@@ -281,6 +281,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("histogram_percentiles", []string{"0.95"})
 	config.BindEnvAndSetDefault("aggregator_stop_timeout", 2)
 	config.BindEnvAndSetDefault("aggregator_buffer_size", 100)
+	config.BindEnvAndSetDefault("aggregator_telemetry_tags_enabled", false) // configure adding the agent container tags to the telemetry metrics sent by the aggregator
```

Comment:
The option name could reflect that it adds container tags (unless it could add other tags as well?). And I think we could call this "basic telemetry" (not a fan of calling it "basic" TBH, but it's to differentiate it from the larger telemetry options, and to avoid mentioning the aggregator directly in the option name, which is a bit of an implementation detail).

```suggestion
	config.BindEnvAndSetDefault("basic_telemetry_add_container_tags", false) // configure adding the agent container tags to the basic agent telemetry metrics (e.g. `datadog.agent.running`)
```

Reply:
yes we're only adding container and pod tags by enabling this, and I yes this must be differentiated from the other telemetry, actually that's why I've put the option here under the aggregator options instead of the `telemetry` config.
thanks for the suggestion 👍